spec/mailers/postmaster_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PostmasterMailer, type: :mailer do
+  let!(:create_admin_permission) do
```

Comment:
you need permissions only if you gonna use private api / create users in the tests with this type of role.
As i see - enough just to have member permission